contracts/staking/AbstractGoodStaking.sol

```diff
@@ -113,19 +118,17 @@ contract AbstractGoodStaking {
      * @param amount tokens to be redeemed.
      */
     function redeem(uint amount) internal virtual{}
-
     /**
      * @dev Redeem invested underlying tokens from defi protocol
      * @dev amount tokens to be redeemed
      * @return token which redeemed from protocol and redeemed amount
```

Comment:
do we expect this to be DAI?
the need to update comment

Reply:
No it's particular token iToken so can be any token and cToken

Reply:
so where is the function that we expect to return intereset in DAI?

Reply:
redeemUnderlying returns interest in cdai which I call in collectUBIInterest

Reply:
see latest comment in compoundstaking redeemUnderlying should probably be redeemUnderlyingToDAI

Reply:
changed